.editorconfig

```diff
@@ -1,26 +1,33 @@
 root = true
 
-[*.cs]
+# Don't use tabs for indentation.
+[*]
 indent_style = space
 end_of_line = lf
-indent_size = 4
-charset = utf-8
 trim_trailing_whitespace = true
+
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
 insert_final_newline = true
-dotnet_sort_system_directives_first = true
+charset = utf-8
 
 [*.{ts,tsx,js}]
-indent_style = space
-end_of_line = lf
 indent_size = 2
 charset = utf-8
-trim_trailing_whitespace = true
 insert_final_newline = true
 
-[.resx]
-indent_style = space
-end_of_line = lf
+
+# Xml config files
+[*.{props,targets,ruleset,config,nuspec,resx,vsixmanifest,vsct,xml,stylecop}]
```

Comment:
Do we have any files with these extensions? Ditto for csx, vb, vbx, and cake

Reply:
We have files belonging to these types `*.{props,targets,config,resx,xml}` but I don't really care what the `indent_size` for them is. I was copying from https://github.com/dotnet/templates/blob/main/.editorconfig because I thought it would be useful to adhere to a canonical format, but I am happy to remove

Reply:
I would guess that the template is set up to deal with old .NET framework projects; you wouldn't be able to use it with most repos targeting Linux deployments (the `[*]` would break any `Makefile`), and it seems geared for projects that may have VB scripts floating around. My recommendation would be to just add the `csharp_style_implicit_object_creation_when_type_is_apparent` directive to the existing structure to avoid breaking all the CRLF baselines.